modules/private-clusters-setting-api-private.adoc

```diff
@@ -38,26 +38,44 @@ ifndef::cpmso-using-azure[]
 *** For AWS, delete the external load balancer. The API DNS entry in the private zone already points to the internal load balancer, which uses an identical configuration, so you do not need to modify the internal load balancer.
 endif::cpmso-using-azure[]
 ifndef::cpmso-using-aws[]
-*** For Azure, delete the `api-internal` rule for the load balancer.
+*** For Azure, delete the public load balancer.
```

Comment:
I'm not sure that this is the correct approach. In Azure, we rely on the public load balancer to provide egress to the internet, I'm not sure what the desired outcome is for these private clusters, but I think they still need egress.

It's possible instead we need to delete some inbound rules, but not the entire load balancer.

Will need to set up an Azure cluster to verify